src/secp256k1.ts

```diff
@@ -15,10 +15,12 @@ function deriveScalar(bytes, discrim?: number) {
     }
     hasher.addU32(i)
     const key = hasher.first256BN()
+    /* istanbul ignore else */
     if (key.cmpn(0) > 0 && key.cmp(order) < 0) {
```

Comment:
Out of curiosity, what does this `if` statement mean in plain English?

Reply:
Good question - I don't know. It originates from over 4 years ago https://github.com/ripple/ripple-keypairs/commit/da77b75751a7de36d0ab09ae8763f81a6f8113a1

Could be something to look into during the audit